lib/logger.js

```diff
@@ -1,23 +1,26 @@
-var winston = require('winston');
+const winston = require('winston');
 
+const logger = new winston.Logger({});
 
-var logger = new (winston.Logger)({});
+const enableLogs = process.env.NODE_ENV === 'development' ||
+	!!process.env.ENABLE_THREADNEEDLE_LOGS;
```

Comment:
I think it's best if environment variables start off with the lib's name, so they're easily identifiable for what libs the flags belong to. So rename to `THREADNEEDLE_ENABLE_LOGS`

Reply:
Yeah good point 👍